### ISSUE TITLE
v2 - Custom Component is duplicated in profile

### ISSUE THREAD

**Issue by manhofmann** [jeffgreco13/filament-breezy#199]
Hello,

using 

`
php artisan make:livewire MyCustomComponent
`

and registering it via method 

BreezyCore::make()
    ->myProfileComponents([MyCustomComponent::class])

seems to show the livewire component twice in profile. The issue seems to be in file src/BreezyCore.php, method:

    public function boot(Panel $panel): void
    {
        if ($this->myProfile) {
            Livewire::component('personal_info', PersonalInfo::class);
            Livewire::component('update_password', UpdatePassword::class);
            $this->myProfileComponents(array_merge([    <----------- appends component
                'personal_info' => PersonalInfo::class,
                'update_password' => UpdatePassword::class
            ], $this->registeredMyProfileComponents)); 

and/or this method:

    public function myProfileComponents(array $components)
    {
        $this->registeredMyProfileComponents = [
            ...$this->registeredMyProfileComponents,
            ...$components,  
        ];

        return $this;
    }

Basically method myProfileComponents gets called twice: once in AppProvider and BreezyCore 

Version: v2.1.1

**Comment by jeffgreco13** [comment]
Fixed in v2.1.2
